tests/progs/test_jubjub.py

```diff
@@ -216,3 +212,26 @@ def test_basic_point_functionality():
             await p1.montgomery_mul(5))
 
     await run_test_prog(_prog, test_preprocessing)
+
+
+@mark.asyncio
+async def test_share_mul(test_preprocessing):
+
+    p = TEST_POINTS[1]
+    multiplier = 7
```

Comment:
this is not a good test... if the share to multiply by is represented by a sequence of secret shared bits, which can potentially take the full range of Fp, approx ~255 bits, then you shouldn't test it just by passing in 3 bits rather than 255